examples/librados/hello_world.cc

```diff
@@ -144,7 +144,7 @@ int main(int argc, const char **argv)
    * now let's read that object back! Just for fun, we'll do it using
    * async IO instead of synchronous. (This would be more useful if we
    * wanted to send off multiple reads at once; see
-   * http://docs.ceph.com/docs/master/rados/api/librados/#asychronous-io )
+   * http://docs.ceph.com/docs/master/rados/api/librados/#asynchronous-io )
```

Comment:
Only if that anchor is spelled correctly too

Reply:
I should have called it out, the anchor in the actual doc is spelled properly (I actually checked this one eagerly, but it was long before I made the PR and I forgot to mention it when annotating the PR).